mpi-proxy-split/mpi_plugin.cpp

```diff
@@ -46,12 +48,16 @@
 #include "jfilesystem.h"
 #include "protectedfds.h"
 #include "procselfmaps.h"
+#ifdef SINGLE_CART_REORDER
```

Comment:
Minor: you can make it a single `#ifdef` block by putting `signal.h, sys/personality.h, sys/mman.h` on top of `stdio.h` and by putting `cartesian.h` right under `sys/stat.h`.